packages/experimental/Expander/windows/ReactNativeExpander/ExpanderView.cpp

```diff
@@ -0,0 +1,255 @@
+#include "pch.h"
+#include "JSValueXaml.h"
+#include "ExpanderView.h"
+#include "ExpanderView.g.cpp"
+
+#include <limits>
+#include <stdexcept>
+
+namespace winrt {
+    using namespace Microsoft::ReactNative;
+    using namespace Windows::Foundation;
+    using namespace Windows::UI::Xaml::Media;
+}
+
+namespace winrt::ReactNativeExpander::implementation {
+
+    ExpanderView::ExpanderView(winrt::IReactContext const& reactContext) : m_reactContext(reactContext) {
+        auto expander = Microsoft::UI::Xaml::Controls::Expander();
+        this->VerticalContentAlignment(Windows::UI::Xaml::VerticalAlignment::Top);
```

Comment:
@asklar Should we use the aliases `xaml::` namespace here? Or is that something that is only necessary in RNW core code, no longer necessary with our update of the default winui version?

Reply:
yes, this should use the appropriate alias so that it will "just work" with WinUI 3.

I'm still a little confused as to the nature of this change overall... I thought the idea was to use react-native-xaml to expose the control to JS, it seems like this change is just hand-rolling duplicated logic we already have in place?

Reply:
I am not sure the aliased types are publicly exported outside of RNW.

Reply:
we expose cppwinrtincludes.h and the ui.*.h headers